playItAgain/css/app.css

```diff
@@ -0,0 +1,88 @@
+body{
+  background-color:orange;
+  font-family: "Monospace";
+}
+
+div{
+  padding: 20px;
+}
+
+#oscillator-list { /* Descendent Selector */
+  list-style-type: none;
+  padding-left: 0px;
+  color:white;
+}
+
+#sawtooth {
+  background-color: #336E91;
+}
+
+#sine {
+  background-color: #783d47;
+}
+
+#triangle {
+  background-color: #3b3040;
+}
+
+#square {
+  background-color: #b85635;
+}
+
+.selected-waveform {
+  outline: 2px solid #FFFFFF; 
+}
+
+.selected-solfege {
+  outline: 2px solid #FFFFFF;
+}
+
+#solfege-list {
+  list-style-type: none;
+  padding-left: 0px;
+  color:white;
+}
+
+#do {
```

Comment:
these names, ids or classes aside, are way way to short and therefore meaningless to other developers. Logger, more specific, more intuitively understandable names, are always better. Never choose short over understandable.